installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_azure_rules.json

```diff
@@ -2515,12 +2515,12 @@
     {
       "ruleId": "Ensure_Web_App_latest_TLS_version",
       "ruleUUID": "Ensure_Web_App_latest_TLS_version",
-      "policyId": "Ensure_Web_App_latest_TLS_version",
+      "policyId": "Enable_latest_TLS_version",
       "ruleName": "Enable WebApp min TLS version is 1.2",
       "targetType": "webapp",
       "assetGroup": "azure",
       "alexaKeyword": "Ensure_Web_App_min_TLS_version_1.2",
-      "ruleParams": "{\"params\":[{\"encrypt\":false,\"value\":\"check-for-Tls-version-rule\",\"key\":\"ruleKey\"},{\"key\":\"description\",\"value\":\"check TLS Protocol Latest Version\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"violationReason\",\"value\":\"webapp min Tls version is not 1.2\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"esSgRulesUrl\",\"value\":\"/azure_webapp/_search\",\"encrypt\":false},{\"key\":\"policyName\",\"value\":\"Enable WebApp min TLS version is 1.2\",\"isValueNew\":true,\"encrypt\":false},{\"encrypt\":false,\"value\":\"medium\",\"key\":\"severity\"},{\"encrypt\":false,\"value\":\"security\",\"key\":\"ruleCategory\"},{\"encrypt\":false,\"value\":\"\",\"key\":\"ruleOwner\"}],\"environmentVariables\":[],\"ruleId\":\"Ensure_Web_App_latest_TLS_version\",\"autofix\":false,\"alexaKeyword\":\"Ensure_Web_App_latest_TLS_version\",\"ruleRestUrl\":\"\",\"targetType\":\"webapp\",\"pac_ds\":\"azure\",\"policyId\":\"Ensure_Web_App_latest_TLS_version\",\"assetGroup\":\"azure\",\"ruleUUID\":\"Ensure_Web_App_latest_TLS_version\",\"ruleType\":\"ManageRule\"}",
+      "ruleParams": "{\"params\":[{\"encrypt\":false,\"value\":\"check-for-Tls-version-rule\",\"key\":\"ruleKey\"},{\"key\":\"description\",\"value\":\"check TLS Protocol Latest Version\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"violationReason\",\"value\":\"webapp min Tls version is not 1.2\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"esSgRulesUrl\",\"value\":\"/azure_webapp/_search\",\"encrypt\":false},{\"key\":\"policyName\",\"value\":\"Enable WebApp min TLS version is 1.2\",\"isValueNew\":true,\"encrypt\":false},{\"encrypt\":false,\"value\":\"medium\",\"key\":\"severity\"},{\"encrypt\":false,\"value\":\"security\",\"key\":\"ruleCategory\"},{\"encrypt\":false,\"value\":\"\",\"key\":\"ruleOwner\"}],\"environmentVariables\":[],\"ruleId\":\"Ensure_Web_App_latest_TLS_version\",\"autofix\":false,\"alexaKeyword\":\"Ensure_Web_App_latest_TLS_version\",\"ruleRestUrl\":\"\",\"targetType\":\"webapp\",\"pac_ds\":\"azure\",\"policyId\":\"Enable_latest_TLS_version\",\"assetGroup\":\"azure\",\"ruleUUID\":\"Ensure_Web_App_latest_TLS_version\",\"ruleType\":\"ManageRule\"}",
```

Comment:
Can we parameterise the TLS version in rule param.
Also instead of TLS version 1.2 in ruleName, please change to latest TLS version